src/entities/movie/hooks/useGetTrendedMovies.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "@tanstack/react-query"
```

Comment:
Let's install this plugin for prettier - https://github.com/trivago/prettier-plugin-sort-imports To make imports order more readable